apps/semver-yeasy/semver-yeasy.sh

```
@@ -135,39 +136,44 @@ calculate-version)
+
+                if [ -z "${gitversion_calc}" ]; then
+                    echo "Error: gitversion_calc turned out to be empty. Please check the configuration file and the command."
+                    exit 1
+                fi
+                
+                # Used for debugging
+                log "gitversion_calc=$($gitversion_calc_cmd 2>&1)"
+                exit_status=$?
+                log "Exit status: $exit_status" >> $GITHUB_OUTPUT
+
+                if [ "${GITHUB_EVENT_NAME}" == "push"]; then
```

Shellcheck (0.10.0)

[error] Couldn't parse this test expression. Fix to allow more checks.

(SC1073)